src/utils.ts

```diff
@@ -660,10 +660,11 @@ export const _register_event = (function () {
 })()
 
 export const _info = {
-    campaignParams: function (): Record<string, any> {
-        const campaign_keywords = 'utm_source utm_medium utm_campaign utm_content utm_term gclid fbclid msclkid'.split(
-            ' '
-        )
+    campaignParams: function (customParams?: string[]): Record<string, any> {
+        const campaign_keywords = 'utm_source utm_medium utm_campaign utm_content utm_term gclid fbclid msclkid'
```

Comment:
While we're at it, can we turns this disgusting string into a plain array?

Reply:
That's a good point. I love how I just ignored it 😆